.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: android
-sudo: required
```

Comment:
the article only says about removing sudo option if it is set to false

Reply:
Yes. So there is no reason to have this. The more default the config, the better.